radio_beam/beam.py

```diff
@@ -40,11 +42,11 @@ def __new__(cls, major=None, minor=None, pa=None, area=None,
 
         # error checking
 
-        # ... given an area make a round beam
+        # ... given an area make a round beam assuming it is Gaussian
         if area is not None:
-            rad = np.sqrt(area/np.pi) * u.deg
-            major = rad
-            minor = rad
+            rad = np.sqrt(area/(2*np.pi)) * u.deg
+            major = rad * SIGMA_TO_FWHM
+            minor = rad * SIGMA_TO_FWHM
```

Comment:
Prior to this change, `area` assumed a top-hat beam, even though everything else about `RadioBeam` assumes a Gaussian beam.